pom.xml

```diff
@@ -35,16 +37,14 @@
     <dependency>
       <groupId>com.esri.geoevent.sdk</groupId>
       <artifactId>geoevent-sdk</artifactId>
-      <version>${project.version}</version>
+      <version>[{project.version},)</version>
```

Comment:
what is the change here?

Reply:
Thank you for pointing it out; there is a typo there - $ omitted - although it still worked, I will fix that. The ${v} => [${v},) just explicitly indicates that future versions are supported, as recommended by Maven. Considering that 10.4.1 is quite an old release, someone perusing the source code for indications of forward compatibility might find that reassuring, even though it does match the default behavior for most cases. I don't insist on it; I just thought it was a good idea.

Reply:
can we keep it simply as  ```<version>${project.version}</version>```?
We don't know which future version might not be supported.